.gitignore

```diff
@@ -62,9 +62,6 @@ build/
 [Ll]ogs/
 out/
 
-# Builds may generate a CCache directory.
-ccache/
```

Comment:
why does this need to change? Does the ACL issue affect the ccache contents? If so, can we run the setfacl command on the existing directory, and avoid issues with moving it?

Reply:
Reverted the change. Shouldn't be needed for this fix.